app/controllers/works_controller.rb

```diff
@@ -48,8 +48,6 @@ def update
     @form = work_form(work)
     if @form.validate(work_params) && @form.save
       work.event_context = { user: current_user }
```

Comment:
Should this line also move into `#after_save` as in the `CollectionsController`?